Content.Shared/Chemistry/Reaction/SharedChemicalReactionSystem.cs

```diff
@@ -13,18 +13,60 @@ namespace Content.Shared.Chemistry.Reaction
 {
     public abstract class SharedChemicalReactionSystem : EntitySystem
     {
-        private IEnumerable<ReactionPrototype> _reactions = default!;
 
+        /// <summary>
+        /// 
+        /// </summary>
         private const int MaxReactionIterations = 20;
 
         [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
         [Dependency] private readonly IRobustRandom _random = default!;
-        [Dependency] protected readonly SharedAdminLogSystem _logSystem = default!;
+        [Dependency] protected readonly SharedAdminLogSystem LogSystem = default!;
+
+        /// <summary>
+        /// 
+        /// </summary>
+        private IDictionary<string, List<ReactionPrototype>> _reactions = default!;
 
         public override void Initialize()
         {
             base.Initialize();
-            _reactions = _prototypeManager.EnumeratePrototypes<ReactionPrototype>();
+            InitializeReactions();
+        }
```

Comment:
This should probably also update when prototypes are reloaded. See for example `SpriteAccessoryManager` and how it uses `_prototypeManager.PrototypesReloaded`

Reply:
Done.